### ISSUE TITLE
Gets the exact number of message backlog for pulsar topic subscription

### ISSUE THREAD

**Issue by xuesongxs** [apache/pulsar#10705]
**Is your feature request related to a problem? Please describe.**
The producer's asynchronous sending interface is batch sending, and a backlog is a batch of messages. To get the number of messages in the subscription backlog, you can only view the number of batches in the backlog, but not the exact number of messages. Expect to support getting the exact number of messages backlog.

**Comment by gaozhangmin** [comment]
Now, version-2.7.2 num of message in backlog is batched.
![image](https://user-images.githubusercontent.com/9278488/121299365-e2264b00-c927-11eb-86a4-503d1077e58e.png)

**Comment by xuesongxs** [comment]
yes, backlog is batched，however, the value of each batch is not fixed, so the accurate number of messages cannot be obtained，



At 2021-06-09 13:38:12, "gaozhangmin" ***@***.***> wrote:

Now, version-2.7.2 num of message in backlog is batched.

**Comment by codelipenghui** [comment]
The issue had no activity for 30 days, mark with Stale label.
